parlai/crowdsourcing/tasks/turn_annotations_static/json/annotation_buckets.json

```diff
@@ -0,0 +1,24 @@
+{
+    "config": {
+        "bucket_0": {
+            "name": "Bucket 0",
+            "description": "this response implies something...0"
+        },
+        "bucket_1": {
+            "name": "Bucket 1",
+            "description": "this response implies something...1"
+        },
+        "bucket_2": {
+            "name": "Bucket 2",
+            "description": "this response implies something...2"
+        },
+        "bucket_3": {
+            "name": "Bucket 3",
+            "description": "this response implies something...3"
+        },
+        "bucket_4": {
+            "name": "Bucket 4",
+            "description": "this response implies something...4"
+        }
+    }
```

Comment:
it will be good to here to add additional key value pair saying the buckets is supposed to be of input type radio or checkbox in react js.

Reply:
This is noted in the parameter description in the blueprint.